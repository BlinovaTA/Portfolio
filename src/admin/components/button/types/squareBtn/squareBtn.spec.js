import squareBtn from "./squareBtn.vue";
import { mount } from "@vue/test-utils";

it("square button snapshot", async () => {
  const mountWrapper = mount(squareBtn);
  expect(mountWrapper.element).toMatchSnapshot();
});