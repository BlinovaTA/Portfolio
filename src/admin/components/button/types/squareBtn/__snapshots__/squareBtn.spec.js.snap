// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`square button snapshot 1`] = `
<button
  class="square-btn-component"
  type="button"
>
  <div
    class="text"
  >
    <div
      class="sign"
    />
     
    <div
      class="title"
    >
      Добавить работу
    </div>
  </div>
</button>
`;
